### PR TITLE
Add Google Analytics support

### PR DESCRIPTION
**Pull request by mercuryseries**
Checklist:

- [ ] Was this PR discussed in a issue? It is recommended to first discuss a new feature into a GitHub issue before opening a PR.
- [ ] Add tests to cover the fixed bug(s) or the new introduced feature(s) (if appropriate).
- [ ] Update the API documentation is a new function is added or an existing one is deleted.

---

### CODE REVIEW

File: docs/css/custom-openqdc.css [hunk 1]
```diff
@@ -1,5 +1,5 @@
 :root {
-    --openqdc-primary: ##201342;
+    --openqdc-primary: #201342;
```
Comment (mercuryseries) [inline docs/css/custom-openqdc.css:2]: small typo

File: mkdocs.yml [hunk 2]
```diff
@@ -136,3 +136,21 @@ plugins:
...
+        - icon: material/emoticon-happy-outline
+          name: This page was helpful
+          data: 1
+          note: >-
+            Thanks for your feedback!
+        - icon: material/emoticon-sad-outline
+          name: This page could be improved
+          data: 0
+          note: >-
+            Thanks for your feedback!
```
Comment (mercuryseries) [inline mkdocs.yml:156]: Copy-paste from https://github.com/polaris-hub/polaris/blob/main/mkdocs.yml#L128-L146

**Comment by FNTwin** [comment]
Thank you Honore, I'll run the pre-commit on my own, repush and merge it

**Review by FNTwin — APPROVED** [review]
LGTM

**Pull request merged**